WPAndroid Integration.md

```diff
@@ -0,0 +1,113 @@
+2. Merge a PR to `develop` which will be published as `<develop>-<commit full SHA1>`
+3. Create a new tag which will be published as `<tag name>`
+
+**How it works:**
+
+CI will run the following commands:
+
+- `npm ci`
+- `npm run bundle:android`
+- `gutenberg/packages/react-native-bridge/android/publish-aztec-and-bridge.sh <version>`
+
+---
+
+### Manually deploy a new version of `react-native-bridge`
+
+_**IMPORTANT:** This should only be used for testing.
+We shouldn't commit a manually deployed version to WPAndroid's `develop` because the process bypasses the guarantees CI would otherwise give us._
+
+- Make sure you run `npm install` or `npm ci` first
+- Run `npm run bundle:android` so the JS bundle is created
+- Run `publish-aztec-and-bridge.sh` script in `react-native-bridge`, giving it a version name such as `locally-built-by-<your_name>-<commit full SHA1>`
+
+**How it works:**
+
+Besides checking a few things, `publish-aztec-and-bridge.sh` script will clean the project, copy the bundled JS file into its assets, publish `react-native-aztec` and use that version to build and publish `react-native-bridge`.
+
+---
+
+### Update WPAndroid's binary version
+
+- Find [the version deployed from CI](#deploy-a-new-version-of-react-native-bridge-from-ci) or [the version you've manually deployed](#manually-deploy-a-new-version-of-react-native-bridge)
+- Update `ext.gutenbergMobileVersion` property in [build.gradle](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/build.gradle) to the new version
+
+In order to test this, make sure `localGutenbergMobilePath` in your `local-builds.gradle` file is commented out as otherwise the binary version will be ignored.
+
+---
+
+## How to
+
+### How to test a gutenberg-mobile PR in WPAndroid
+
+Assuming that there is no open WPAndroid PR yet:
+
+1. Setup WPAndroid to [work with local gutenberg-mobile checkout](#work-with-local-gutenberg-mobile-checkout)
+2. Switch `gutenberg-mobile` to the branch you're testing
+3. Run the app
+
+---
+
+### How to share a WIP WPAndroid APK
+
+1. Open a `gutenberg-mobile` PR which will [publish a new version](#deploy-a-new-version-of-react-native-bridge-from-ci) or [manually publish a new version](#manually-deploy-a-new-version-of-react-native-bridge)
+2. Open a WPAndroid PR [updating the binary version](#update-wpandroids-binary-version)
+3. Peril will publish a comment to the WPAndroid PR with a link to the APK
+
+Alternatively, once the `react-native-bridge` version is published, we can locally create the APK.
+**Note that there is no way to use composite build to create a WPAndroid APK that can be shared. This is because composite build only works with the local metro server.**
+
+---
+
+### How to checkout gutenberg-mobile to deployed binary version
+
+Whenever the CI publishes `react-native-bridge`, it will either include the full commit hash or the tag in its name.
+So, we can checkout simply checkout the commit has or the tag.
```

Comment:
has -> hash

Reply:
🤦 Fixed in 6a5ca7effc502ea798275d8ced92a54755d14942.